code/datums/elements/turf_transparency.dm

```diff
@@ -37,8 +33,10 @@
 ///Updates the viscontents or underlays below this tile.
 /datum/element/turf_z_transparency/proc/update_multi_z(turf/our_turf)
 	var/turf/below_turf = our_turf.below()
-	if(!below_turf)
-		our_turf.vis_contents.len = 0
+	if(below_turf)
+		our_turf.vis_contents += below_turf
+	else
+		our_turf.vis_contents.Cut()
```

Comment:
I'm pretty sure setting the len to 0 is a slight microopt and this is probably a heavy proc